### DISCUSSION TITLE
Is it possible to run a script on anothter extension page?

### DISCUSSION THREAD

**Discussion by GlassGruber** [violentmonkey/violentmonkey#1805] [Q&A] [answered]
I'd like to run a script in Firefox on another extension internal page, one of those accessible via urls like `moz-extension://<uuid>/index.html`. 

Actually the popup says

> Violentmonkey cannot run userscripts in this page (common examples: browser UI, an extension, blocked via policies, a search engine site in Opera)

which is an expected and welcome default ofc.

So is it possible via some specific permission or `about:config` rule to allow VM to run scripts on a specific extension page ?
(I'm more interested for Firefox browser but I guess this could apply to Chromium too)

Thank you for any suggestions!

**Comment by tophf** [comment, marked as answer]
No, Violentmonkey doesn't provide this feature and AFAIK there's no way to implement it in Firefox.

**Reply by GlassGruber** [reply]
I see, thank you for your answer!
